openedx/features/edly/models.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.sites.models import Site
+from django.core.validators import RegexValidator
+from django.db import models
+from model_utils.models import TimeStampedModel
+from organizations.models import Organization
+
+EDLY_SLUG_VALIDATOR = RegexValidator(r'^[0-9a-z-]*$', 'Only small case alphanumeric and hyphen characters are allowed.')
+
+
+class EdlyOrganization(TimeStampedModel):
+    """
+    EdlyOrganization model for Edly.
+    """
+    name = models.CharField(max_length=255)
+    slug = models.CharField(max_length=50, unique=True, validators=[EDLY_SLUG_VALIDATOR])
```

Comment:
@asadmanzoor93 Why didn't we use the Django default slug field here?
https://docs.djangoproject.com/en/3.0/ref/models/fields/#slugfield